rdr_service/resource/constants.py

```diff
@@ -46,6 +46,12 @@ class SchemaID(IntEnum):
     workbench_institutional_affiliation = 4010
     workbench_workspace = 4020
     workbench_workspace_users = 4030
+    workbench_researcher_race = 4040
```

Comment:
questions for @robabram :   What is the trigger for needing a distinct SchemaID value when it comes to "sub"-schemas?  Is the expectation that any nested field in  a defined resource Schema will always need its own resource_type / SchemaID and should be retrievable  via the resource API?

Also:  where do we have the documentation for populating newly defined content in the RDR  `resource_type` and `resource_schema` tables when we  update these lists?

Reply:
Defining a schema id value is a bit arbitrary, but I wanted to be able to identify a schema id group by the left most digit(s) of the schema id.